cmd/serve.go

```diff
@@ -74,6 +74,7 @@ func (s *ServerService) GetSchedulerJobRegistrars(ctx context.Context, serveOpts
 			CrashTrackerClient:       serveOpts.CrashTrackerClient.Clone(),
 		}),
 		scheduler.WithAPAuthEnforcementJob(apAPIService, serveOpts.MonitorService, serveOpts.CrashTrackerClient.Clone()),
+		scheduler.WithPatchAnchorPlatformTransactions(apAPIService, models),
```

Comment:
💡 There is another occasion where we patch the AP, so you need to be more explicit here that this is a PATCH related to completion.

```suggestion
		scheduler.WithPatchAnchorPlatformTransactionsCompletion(apAPIService, models),
```